synapseutils/sync.py

```diff
@@ -719,7 +749,8 @@ def generateManifest(syn, allFiles, filename, provenance_cache=None) -> None:
         syn: A Synapse object with user's login, e.g. syn = synapseclient.login()
         allFiles: A list of File Entity objects on Synapse (can't be Synapse IDs)
         filename: file where manifest will be written
-        provenance_cache: an optional dict of known provenance dicts keyed by entity ids
+        provenance_cache: an optional dict of known provenance dicts keyed by entity
+        ids
```

Comment:
Can you check that this renders correctly on the docs page. This may need to be indented once is properly show up.

Reply:
It does not. Fixed in the latest commit.